packages/plugin-compat/extra/resolve/normalize-options.js

```diff
@@ -88,14 +103,16 @@ module.exports = function (_, opts) {
   // the code is compatible with both `resolve` 1.9+ and `resolve` 1.15+
   let isInsideIterator = false;
 
-  opts.packageIterator = function (request, basedir, getCandidates, opts) {
-    isInsideIterator = true;
-    try {
-      return packageIterator(request, basedir, getCandidates, opts);
-    } finally {
-      isInsideIterator = false;
-    }
-  };
+  if (!opts.__skipPackageIterator) {
```

Comment:
What's the point of this 'option'? It doesn't seem to be set in anything else than the tests 🤔

Reply:
It's only for the tests, it's so that I can make sure versions of resolve before the PackageIterator was introduced works.